runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -232,7 +235,7 @@ public void onGameStateChanged(final GameStateChanged event)
 	 * @param itemID item id
 	 * @return item price
 	 */
-	public int getItemPrice(int itemID)
+	public long getItemPrice(int itemID)
```

Comment:
these both methods can be int, because both GE and HA price can never exceeed max int